// The Firmament project
// Copyright (c) 2011-2012 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// Task information RPC messages.

syntax = "proto3";

package firmament;

import "base/task_desc.proto";

message TaskInfoRequestMessage {
  uint64 task_id = 1;
  string requesting_resource_id = 2;
  string requesting_endpoint = 3;
  uint32 task_pid = 4;
}

message TaskInfoResponseMessage {
  uint64 task_id = 1;
  TaskDescriptor task_desc = 2;
}
